src/integrationTest/groovy/CompilationAndGeneration.groovy

```diff
@@ -18,8 +18,8 @@ public class CompilationAndGeneration {
         ImmutableMultimap.Builder<String, ArrayList<String>> builder = ImmutableMultimap.builder();
         builder.put(DIRECTORIES_ROOT + "subproj/build/classes/generatedSource", ["Subproj.class", "Subproj2.class"])
         builder.put(DIRECTORIES_ROOT + "sub2/build/classes/generatedSource", ["Sub2.class", "Sub22.class"])
-        builder.put(DIRECTORIES_ROOT + "subproj/generated/java", ["Subproj.java", "Subproj2.java"])
-        builder.put(DIRECTORIES_ROOT + "sub2/generated/java", ["Sub2.java", "Sub22.java"])
+        builder.put(DIRECTORIES_ROOT + "subproj/customPath/generated/java", ["Subproj.java", "Subproj2.java"])
```

Comment:
I agree we should be testing both. I chose to test the custom fields here because I think those are higher priority. The only way I can see to add another set of tests would be to add another test project. That would add a lot of duplicate code and take time, and I really want to focus on getting the compiler jar built. Also, I'm sure there is a smarter way to add another set of tests. 

What I'd like to do is update #8 so that it's clear that more tests need to be added. If I have time I'll get to it after the umple jar, if not then then next person can tackle it.